static/monorail.yml

```diff
@@ -650,34 +650,12 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
-  /workflows/library:
-    get:
-      summary: |
-        List all workflows available to run
-      description: |
-        List all workflows available to run
-      tags:
-        - workflow
-        - get
-      responses:
-        200:
-          description: |
-            List all workflows available to run
-          schema:
-            type: array
-            items:
-              type: object
-        default:
-          description: Unexpected error
-          schema:
-            $ref: '#/definitions/Error'
-
   /workflows/library/{injectableName}:
     get:
       summary: |
-        List all workflows available to run
+        Get a workflow definition
       description: |
-        List all workflows available to run
+        Get a workflow definition. To list all available workflows, use the '*' as a wildcard for the name.
```

Comment:
actually it is - @benbp made changes several months ago, but it didn't propagate to this or the documentation, so this is the doc catchup. The 2.0 API made it work as you would normally expect, without a wildcard, but the 1.1 API had limited based on the URI structuring and path rendering that made that infeasible.